packages/design-system/src/components/Modal/Modal.stories.tsx

```diff
@@ -1,39 +1,249 @@
+				flutes, lightly prick the base with a fork, then chill for 10 mins.
+			</p>
+
+			<p>
+				<b>STEP 5</b>
+				<br />
+				Put a baking sheet in the oven and heat oven to 200C/fan 180C/gas 6. Line pastry case with
+				foil, shiny side down, fill with dry beans and bake on the hot sheet for 15 mins.
+			</p>
+
+			<p>
+				<b>STEP 6</b>
+				<br />
+				Remove foil and beans and bake for 4-5 mins more until the pastry is pale golden. If you
+				notice any small holes or cracks, patch up with pastry trimmings. You can make up to this
+				point a day ahead.
+			</p>
+
+			<p>
+				<b>STEP 7</b>
+				<br />
+				While the pastry cooks, prepare the filling. Heat a small frying pan, tip in 200g lardons
+				and fry for a couple of mins. Drain off any liquid that comes out, then continue cooking
+				until the lardons just start to colour, but aren’t crisp. Remove and drain on paper towels.
+			</p>
+
+			<p>
+				<b>STEP 8</b>
+				<br />
+				Cut three quarters of the 50g gruyère into small dice and finely grate the rest. Scatter the
+				diced gruyère and fried lardons over the bottom of the pastry case.
+			</p>
+
+			<p>
+				<b>STEP 9</b>
+				<br />
+				Using a spoon, beat 200ml crème fraîche to slacken it then slowly beat in 200ml double
+				cream. Mix in 3 well beaten eggs. Season (you shouldn’t need much salt) and add a pinch of
+				ground nutmeg. Pour three quarters of the filling into the pastry case.
+			</p>
+
+			<p>
+				<b>STEP 10</b>
+				<br />
+				Half-pull the oven shelf out and put the flan tin on the baking sheet. Quickly pour the rest
+				of the filling into the pastry case – you get it right to the top this way. Scatter the
+				grated cheese over the top, then carefully push the shelf back into the oven.
+			</p>
+
+			<p>
+				<b>STEP 11</b>
+				<br />
+				Lower the oven to 190C/fan 170C/gas 5. Bake for about 25 mins, or until golden and softly
+				set (the centre should not feel too firm).
+			</p>
+
+			<p>
+				<b>STEP 12</b>
+				<br />
+				Let the quiche settle for 4-5 mins, then remove from the tin. Serve freshly baked, although
+				it’s also good cold.
+			</p>
+
+			<div style={{ width: '50%', margin: '2rem auto 0' }}>
+				<img
+					src="https://img-3.journaldesfemmes.fr/csLNATf47C8IYJxFtQ4S-o1t0kw=/800x600/smart/5a1c637d7ef0426784dad14c29aaff55/recipe-jdf/10025089.jpg"
+					alt="The quiche lorraine"
+					// width: '50'
+				/>
+			</div>
+		</div>
+	);
+}
+
+export const Basic: ComponentStory<typeof Modal> = props => (
+	<ModalStory {...props} header={{ title: 'Basic modal' }}>
+		<p>A basic modal with only a title and a text content.</p>
+	</ModalStory>
+);
+
+export const WithDisclosure: ComponentStory<typeof Modal> = props => (
 	<Modal
-		baseId=""
+		{...props}
+		header={{ title: 'With disclosure' }}
 		disclosure={
-			<ButtonPrimary onClick={action('Clicked disclosure')}>Send invitation email</ButtonPrimary>
+			<ButtonPrimary data-test="modal-disclosure" onClick={() => {}}>
```

Comment:
```suggestion
			<ButtonPrimary data-testid="modal-disclosure" onClick={() => {}}>
```
IIRC (@frassinier) for things that would come from the project's side, then we use `data-testid`. This mimics what would happen in a product. 

https://github.com/Talend/ui/blob/master/packages/design-system/adr/data-test.md